File: src/entites/commande.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package entites;

/**
 *
 * @author HAMZA
 */
public class commande {
    private int idCommande;
    private int idClient;
    private float montantPanier;

    public commande() {
    }

    public commande(int idClient, float montantPanier) {
        
        this.idClient = idClient;
        this.montantPanier = montantPanier;
    }

    public int getIdCommande() {
        return idCommande;
    }

    public int getIdClient() {
        return idClient;
    }

    public float getMontantPanier() {
        return montantPanier;
    }

    public void setIdCommande(int idCommande) {
        this.idCommande = idCommande;
}

    public void setIdClient(int idClient) {
        this.idClient = idClient;
    }

    public void setMontantPanier(float montantPanier) {
        this.montantPanier = montantPanier;
    }

    @Override
    public String toString() {
        return "commande{" + "idCommande=" + idCommande + ", idClient=" + idClient + ", montantPanier=" + montantPanier + '}';
    }
    
    
    
}

File: src/Entity/Commande.php
<?php

namespace App\Entity;

use Doctrine\ORM\Mapping as ORM;

/**
 * Commande
 *
 * @ORM\Table(name="commande", indexes={@ORM\Index(name="idClient", columns={"idClient"})})
 * @ORM\Entity
 */
class Commande
{
    /**
     * @var int
     *
     * @ORM\Column(name="idCommande", type="integer", nullable=false)
     * @ORM\Id
     * @ORM\GeneratedValue(strategy="IDENTITY")
     */
    private $idcommande;

    /**
     * @var int
     *
     * @ORM\Column(name="montantPanier", type="integer", nullable=false)
     */
    private $montantpanier;

    /**
     * @var \Personne
     *
     * @ORM\ManyToOne(targetEntity="Personne")
     * @ORM\JoinColumns({
     *   @ORM\JoinColumn(name="idClient", referencedColumnName="id")
     * })
     */
    private $idclient;

    public function getIdcommande(): ?int
    {
        return $this->idcommande;
    }

    public function getMontantpanier(): ?int
    {
        return $this->montantpanier;
    }

    public function setMontantpanier(int $montantpanier): self
    {
        $this->montantpanier = $montantpanier;

        return $this;
    }

    public function getIdclient(): ?Personne
    {
        return $this->idclient;
    }

    public function setIdclient(?Personne $idclient): self
    {
        $this->idclient = $idclient;

        return $this;
    }

    public function __toString() {
        return (string)($this->idclient);
    }


}
